setup.py

```diff
@@ -14,7 +14,7 @@
 __version__ = version["__version__"]
 
 install_requires = [
-    "openai>=1,<1.5",  # a temporary fix for breaking changes in 1.5
+    "openai>=1.3,<1.5",
```

Comment:
shouldn't we remove "<1.5" if the PR is supposed to support openai>=1.5?

Reply:
Correct. Somehow this might have been a merge error

Reply:
Yes, that's probably due to merge. I'll run tests again against the latest 1.6.1 and create a new PR.

Reply:
#1071 has been created by @ekzhu and merged.